bin/samba.py

```diff
@@ -34,8 +34,8 @@ def main(data: None | str | pathlib.Path = None, tunnel: bool = False) -> None:
     env["USERID"] = str(os.getuid())
     env["GROUPID"] = str(os.getgid())
     assert env["AXIS_DEVICE_IP"]
-    assert env["AXIS_DEVICE_PASS"]
-    assert env["AXIS_DEVICE_USER"]
+    env["AXIS_DEVICE_PASS"] = env["AXIS_DEVICE_PASS"] or "pass"
```

Comment:
Well spotted, I had forgotten about this. However, `env[...]` will raise a `KeyError` if the variable is not set, so what we need to do is `env.get("AXIS_DEVICE_PASS", "pass")`

Reply:
Or better yet, `env.setdefault("AXIS_DEVICE_PASS", "pass")`